.eslintrc

```diff
@@ -1,6 +1,10 @@
 {
   "extends": [
-    "okonet"
+    "okonet",
```

Comment:
My preset already has prettier so no need in this I guess. I might need to update it to the latest version, though.